supplements/sword-coast-adventurers-guide/classes/monk-longdeath.xml

```diff
@@ -45,7 +45,7 @@
 			<p>At 6th level, you gain the ability to unsettle or terrify those around you as an action, for your soul has been touched by the shadow of death. When you take this action, each creature within 30 feet of you that can see you must succeed on a Wisdom saving throw or be frightened of you until the end of your next turn.</p>
 		</description>
 		<sheet action="Action">
-			<description>Each creature within 30 feet of you that can see you must succeed on a {{ki:dc} Wisdom saving throw or be frightened of you until the end of your next turn.</description>
+			<description>Each creature within 30 feet of you that can see you must succeed on a {{ki:dc}} Wisdom saving throw or be frightened of you until the end of your next turn.</description>
```

Comment:
Maybe change it to, "succeed on a Wisdom saving throw (DC {{ki:dc}}) or" or "succeed on a DC {{ki:dc}} Wisdom saving throw or"

Reply:
yeah, agree, because right now it would read something like "you must succeed on a 15 Wisdom saving throw".
I would change it to "must succeed on a DC{{ki:dc}} Wisdom saving throw"